doc/api/next_api_changes/2019-08-12-KS.rst

```diff
@@ -0,0 +1,8 @@
+Changes in parameter names
```

Comment:
this needs to go to the main next_api_changes/behavior.rst (or deprecations.rst?) file.